### ISSUE TITLE
FUN gene ID numbering seems inconsistent

### ISSUE THREAD

**Issue by DaRinker** [nextgenusfs/funannotate#898]
I'm seeing inconsistent behavior in the default gene numbering in my annotations. For example:
`
$ grep FUN_ DTO15.proteins.fa | head
>DTO15_FUN_000001-T1 locus=DTO15_FUN_000001
>DTO15_FUN_000002-T1 locus=DTO15_FUN_000002
>DTO15_FUN_000003-T1 locus=DTO15_FUN_000003
>DTO15_FUN_000004-T1 locus=DTO15_FUN_000004
>DTO15_FUN_000005-T1 locus=DTO15_FUN_000005
>DTO15_FUN_000006-T1 locus=DTO15_FUN_000006
>DTO15_FUN_000007-T1 locus=DTO15_FUN_000007
>DTO15_FUN_000008-T1 locus=DTO15_FUN_000008
>DTO15_FUN_000009-T1 locus=DTO15_FUN_000009
>DTO15_FUN_000010-T1 locus=DTO15_FUN_000010

$ grep FUN_ DTO15.proteins.fa | tail
>DTO15_FUN_12106-T1 locus=DTO15_FUN_12106
>DTO15_FUN_12107-T1 locus=DTO15_FUN_12107
>DTO15_FUN_12108-T1 locus=DTO15_FUN_12108
>DTO15_FUN_12109-T1 locus=DTO15_FUN_12109
>DTO15_FUN_12110-T1 locus=DTO15_FUN_12110
>DTO15_FUN_12111-T1 locus=DTO15_FUN_12111
>DTO15_FUN_12112-T1 locus=DTO15_FUN_12112
>DTO15_FUN_12113-T1 locus=DTO15_FUN_12113
>DTO15_FUN_12114-T1 locus=DTO15_FUN_12114
>DTO15_FUN_12115-T1 locus=DTO15_FUN_12115

`
The 5-digit gene ID's (without the leading zero) are odd. Moreover, I ALSO have some gene IDs of the formant  "DTO15_FUN_011977-T1" (so it's not just that the leading zeros get dropped when the number crosses 10000...)

This makes the output sometimes hard to parse.  Will I be messing anything up if I attempt to add back in any "missing" zeros using a custom script? (i.e. is there some rhyme/reason to this behavior that I would be tampering with?_

Thank you

**Comment by nextgenusfs** [comment]
Not sure how this is possible. Except use only a single underscore and it should pad to the same number of digits every time. So if you pass --name DTO15 then it will generate the gene names with the locus tag of DTO15 and then an underscore followed by sequential numbering.

**Comment by DaRinker** [comment]
I just checked my output directories.  This phenomena is NOT present in the predict_results outputs but it IS present in all the annotate outputs (_results and _misc). I can trace the same gene going from having the zero padded value to losing the zero. Does that narrow down the possibilities at all?

**Comment by nextgenusfs** [comment]
Did you run `funannotate update`?

**Comment by DaRinker** [comment]
Yes, I  did run funanotate update. (My ordered list of commands is above).

I ran the same analysis over 16 genomes and they all show some of these odd ball numberings.

I'm using a singularity image (if that could be relevant)

**Comment by nextgenusfs** [comment]
Ah yes, sorry I was responding on my phone and with a toddler yelling for my attention at me didn't see the whole post.

**Comment by DaRinker** [comment]
What I'm hearing is that this is not intended behavior and that the missing padding is not meant to signify anything. 

If that's the correct I will go back an add in the missing zeros post-annotation.
